sound/soc/sof/pcm.c

```diff
@@ -323,6 +323,7 @@ static int sof_pcm_trigger(struct snd_pcm_substream *substream, int cmd)
 	struct sof_ipc_stream stream;
 	struct sof_ipc_reply reply;
 	bool reset_hw_params = false;
+	bool stop_sequence = false;
```

Comment:
@xiulipan minor but the name is not intuitive. Why not call it trigger_first or ipc_first or something along those lines?

Reply:
agree with @ranj063, need to find a better name